.scrutinizer.yml

```diff
@@ -1,19 +1,19 @@
 filter:
     excluded_paths:
         - 'vendor/*'
-        - 'tests/*'
 before_commands:
     - 'composer install'
 tools:
     php_analyzer: true
     php_mess_detector: true
     php_code_sniffer:
         config:
-            standard: PSR1
+            standard: PSR2
     sensiolabs_security_checker: true
     php_loc:
         excluded_dirs:
             - vendor
-            - tests
     php_pdepend: true
     php_sim: true
+build_failure_conditions:
+  - 'issues.label("coding-style").new.exists'
```

Comment:
After scrutinizer runs, there may be other items that we need to add to this.

Reply:
scrutinizer finished, so we can add the following build failure conditions
```
    - 'elements.rating(< B).new.exists'                                                                                 
    - 'project.metric("scrutinizer.quality", < 9)' 
```
The [results](https://scrutinizer-ci.com/g/traderinteractive/memoize-php/inspections/48835c80-7c12-406b-b720-e18c666538ff) did uncover 4 major issues. It would be nice if you could fix those

Reply:
done